### DISCUSSION TITLE
Mine selection removal

### DISCUSSION THREAD

**Discussion by Honilno** [voxeltycoon/issues#473] [General]
**Is your feature request related to a problem? Please describe.**
Not needed selection of the mine type

**Describe the solution you'd like**
Instead of selecting proper mine, I should auto select type based on resource which is under cursor.

**Additional context**
It is reduction of few clicks

**Comment by victorleblais** [comment]
I'm not sure this will be worked on since you can have multiple tiers of mines for the same type of resources (different building size and mining capacity) so an "auto-select" can't work.
